docs/en/stack/monitoring/production.asciidoc

```diff
@@ -47,94 +81,45 @@ POST /_xpack/security/user/remote_monitor
-+
---
-To add a CA certificate to an {es} node's trusted certificates, you
-can specify the location of the PEM encoded certificate with the
-`certificate_authorities` setting:
-
-[source,yaml]
---------------------------------------------------
-xpack.monitoring.exporters:
-  id1:
-    type: http
-    host: ["https://es-mon1:9200", "https://es-mon2:9200"]
-    auth:
-      username: agent-user
-      password: password
-    ssl:
-      certificate_authorities: [ "/path/to/ca.crt" ]
---------------------------------------------------
-
-Alternatively, you can configure trusted certificates using a truststore
-(a Java Keystore file that contains the certificates):
-
-[source,yaml]
---------------------------------------------------
-xpack.monitoring.exporters:
-  id1:
-    type: http
-    host: ["https://es-mon1:9200", "https://es-mon2:9200"]
-    auth:
-      username: remote_monitor
-      password: changeme
-    ssl:
-      truststore.path: /path/to/file
-      truststore.password: password
---------------------------------------------------
+. Configure your production cluster to collect data and send it to the 
+monitoring cluster. 
 
---
+** beta[] To use {metricbeat}, see 
+{ref}/configuring-metricbeat.html[Monitoring {es} with {metricbeat}]. This option 
+is available in 6.5 and later versions. 
+
+** To use HTTP exporters, see {ref}/configuring-monitoring.html[Configuring monitoring in {es}].
 
-. Restart {es} on the nodes in your production cluster.
+. (Optional)
+{logstash-ref}/configuring-logstash.html[Configure {ls} to collect data and send it to the monitoring cluster]. 
 +
 --
-TIP: You may want to temporarily {ref}/modules-cluster.html[disable shard
-allocation] before you restart your nodes to avoid unnecessary shard
-reallocation during the install process.
+NOTE: You must configure HTTP exporters in the production cluster to route this 
+data to the monitoring cluster. It cannot be accomplished by using {metricbeat}. 
 
 --
 
-. To verify your {monitoring} installation, point your web browser at your {kib}
-host, and select **Monitoring** from the side navigation. When security is enabled,
-to view the monitoring dashboards you must log in to {kib} as a user who has
-both the `kibana_user` and `monitoring_user` roles. For example:
-+
---
-[source,js]
---------------------------------------------------
-POST /_xpack/security/user/stack-monitor
-{
-  "password" : "changeme",
-  "roles" : [ "kibana_user", "monitoring_user" ]
-}
---------------------------------------------------
-// CONSOLE
+. (Optional) Configure {kib} to collect data and send it to the monitoring cluster:
 
-//image:images/monitoring.jpg["Monitoring",link="images/monitoring.jpg"]
---
+** beta[] To use {metricbeat}, see 
+{kibana-ref}/monitoring-metricbeat.html[Monitoring {kib} with {metricbeat}]. This 
+option is available in 6.4 and later versions. 
+
+** To use HTTP exporters, see 
+{kibana-ref}/monitoring-xpack-kibana.html[Configuring monitoring in {kib}].
```

Comment:
Should we update this link to `{kibana-ref}/monitoring-kibana.html`?

Reply:
Good catch, I've updated it!